testsuite/features/step_definitions/salt_steps.rb

```diff
@@ -689,11 +689,8 @@ def pillar_get(key, minion)
   else
     node.run('zypper --non-interactive remove -y salt salt-minion', false)
   end
-  node.run('rm -Rf /var/cache/salt/minion /var/run/salt /var/log/salt /etc/salt', false)
+  node.run('rm -Rf /var/cache/salt/minion /var/run/salt /var/log/salt /etc/salt /var/tmp/.root*', false)
```

Comment:
As far as I understood here https://github.com/SUSE/spacewalk/issues/13336 we also need to remove this folder `/root/salt`

Reply:
I prefer not to interfeer with that card...

Reply:
no, i don't know that for sure, we are improving this cleanup little by little - sorry, but this is trial and error

the problem is i don't know the ubuntu issue, but @ktsamis does, so there's less risk to do it wrong if he does rather than me